config.go

```diff
@@ -1 +1,105 @@
 package main
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	flags "github.com/btcsuite/go-flags"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	defaultConfigFilename = "lnwallet.conf"
```

Comment:
Config file should instead be: `"lnd.conf"`. `lnwallet` has it's own config struct, be we'll only require users to fill out a single concrete configuration file.